Sergey/Medium/index.html

```diff
@@ -48,137 +50,125 @@ <h1 class="article_header">Where good ideas find you</h1>
             <div class="items_items">
                 <div class="items_number">
                     <div class="number">
-                        <span>01</span>
+                        <p>01</p>
```

Comment:
do this
![do thsis](https://user-images.githubusercontent.com/49835619/97781382-e9b89100-1b9b-11eb-93d7-911f02875a50.png)

Reply:
> do this
> ![do thsis](https://user-images.githubusercontent.com/49835619/97781382-e9b89100-1b9b-11eb-93d7-911f02875a50.png)
Realy. lol)